src/SFA.DAS.IdentifyDataLocks.Web/Infrastructure/ProviderService.cs

```diff
@@ -1,25 +1,88 @@
-using SFA.DAS.Providers.Api.Client;
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using SFA.DAS.Http;
 
 namespace SFA.DAS.IdentifyDataLocks.Web.Infrastructure
 {
     public class ProviderService
     {
-        private readonly IProviderApiClient providerApiClient;
+        private readonly IRoatpService providerApiClient;
 
-        public ProviderService(IProviderApiClient providerApiClient) =>
+        public ProviderService(IRoatpService providerApiClient) =>
             this.providerApiClient = providerApiClient;
 
         public string GetProviderName(long ukprn)
```

Comment:
I would make this `async Task<string>`

and await the async call from `providerApiClient.Get(ukprn)`